src/Apps/Collect/CollectionApp.tsx

```diff
@@ -35,8 +37,10 @@ export class CollectionApp extends Component<CollectionAppProps> {
   }
 
   render() {
-    const { collection } = this.props
+    const { collection, user } = this.props
     const { title, slug, headerImage, description, artworks } = collection
+    const showCollectionHubs =
+      collection.linkedCollections.length > 0 && user && user.type === "Admin"
```

Comment:
There is a util named `userIsAdmin` that can be used to determine if the user is an admin - https://github.com/artsy/reaction/blob/master/src/Utils/user.ts#L34. I know it doesn't replace much code, but it has always felt better to me to use a common function instead of having `user && user.type === "Admin"` throughout the code. If we ever change the way we determine if the user is an admin, we'd have a _ton_ of places to change it.

Reply:
This makes sense!